primer-miso/src/Primer/Miso.hs

```diff
@@ -586,32 +586,36 @@ viewNodeData position dimensions edges node = case node.opts of
           Clay.width $ Clay.px $ realToClay dimensions.x
           Clay.height $ Clay.px $ realToClay dimensions.y
       ]
-  _ ->
-    div_
-      ( [ class_ "node"
-        , class_ case node.level of
+  _ -> div_
```

Comment:
Should we add a comment here about why this is necessary?

Reply:
Yes, probably. I've added to #1348.